core/src/components/range/test/range-events.e2e.ts

```diff
@@ -10,15 +10,28 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
       /**
        * The mouse events are flaky on CI
        */
-      test.fixme('should emit start/end events', async ({ page }, testInfo) => {
-        await page.setContent(`<ion-range value="20"></ion-range>`, config);
+      test.fixme('should emit start/end events', async ({ page }) => {
```

Comment:
I know this is already merged, but just a general question if `test.fixme` should be removed as part of these changes?

Reply:
Range is still flaky so the `fixme` was left in until a fix is implemented.